### PR TITLE
 Added daily archive 

### PR DESCRIPTION
**Pull request by core23**
I am targeting this branch, because this is BC if not tag is created before merging this.

## Changelog

```markdown
### Added
- Added a daily archive
```

## Subject

There is a monthly and a yearly archive. This PR adds a daily archive.

### CODE REVIEW

File: src/Action/DailyPostArchiveAction.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+     * @param string $month
+     * @param string $day
+     *
+     * @return Response
+     */
+    public function __invoke(Request $request, $year, $month, $day)
+    {
+        $date = $this->getPostManager()->getPublicationDateQueryParts(sprintf('%d-%d-%d', $year, $month, $day), 'day');
+
+        if ($seoPage = $this->getSeoPage()) {
```
Comment (greg0ire) [inline src/Action/DailyPostArchiveAction.php]: Early return?

Comment (core23) [reply src/Action/DailyPostArchiveAction.php]: Can you elaborate?

Comment (greg0ire) [reply src/Action/DailyPostArchiveAction.php]: Nevermind, it reviewed too quickly, an early return is not going to be better

File: src/Action/DailyPostArchiveAction.php [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+                    '%date%' => $this->dataTimeHelper->formatDate($date),
+                ]))
+                ->addMeta('property', 'og:description', $this->trans('archive_day.meta_description', [
+                    '%title%' => $this->getBlog()->getTitle(),
+                    '%year%' => $year,
+                    '%month%' => $this->dateTimeHelper->format($date, 'MMMM'),
+                    '%day%' => $day,
+                    '%date%' => $this->dataTimeHelper->formatDate($date),
+                    '%description%' => $this->getBlog()->getDescription(),
+                ]));
```
Comment (greg0ire) [inline src/Action/DailyPostArchiveAction.php]: Why not use a variable to avoid title and description repetition?

**Comment by kunicmarko20** [comment]
I approved because I don't want to be a blocker, but I still think we need tests.

File: tests/DailyPostArchiveActionTest.php [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+            'date' => $dataParams,
+        ], 1)
+            ->willReturn($this->prophesize(PagerInterface::class));
+
+        $twig = $this->prophesize(Environment::class);
+        $twig->render('@SonataNews/Post/archive.html.twig', Argument::any())
+            ->willReturn('HTML CONTENT');
+
+        $container = new Container();
+        $container->set('twig', $twig->reveal());
```
Comment (greg0ire) [inline tests/DailyPostArchiveActionTest.php]: 👍 this avoids the mistake I discovered in my phpstan PR

**Pull request merged**